emission/core/wrapper/suggestion_sys.py

```diff
@@ -200,16 +208,16 @@ def find_destination_business(lat, lon):
     # print(return_address_from_location_google(location))
     # print(len(return_address_from_location_google(location)))
     #IF RETURN_ADDRESS_FROM_LOCATION HAS A BUSINESS LOCATION ATTACHED TO THE ADDRESS
-    try:
+    # try:
```

Comment:
why is this commented out?

Reply:
@valin1 no response to this, code still in place?!

Reply:
oh this was commented out bc i thought it was defaulting to google api

Reply:
i will uncomment